test/integration/server_test.go

```diff
@@ -28,6 +28,7 @@ const (
 
 	serviceURL    = "app:4443"
 	statusURL     = "app:8080"
+	monitoringURL = "app:4269"
```

Comment:
nit: monitoring more feels like prometheus/ metrics stuff. Should we rename to profilerURL or something like that instead?

Reply:
Definitely, much better. Addressed, finally!